frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -105,6 +105,7 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
             showHeader={false}
             pagination={false}
             rowClassName=""
+            rowKey={i => 1}
```

Comment:
I'm not extremely sure of this...

Reply:
It's only for the header, and this is unique in the component